### PR TITLE
new branch name and moved to redux-saga + connected-react-router

### PR DESCRIPTION
**Pull request by joelb01-github**
- changed PR name to follow convention
- moved from `redux-thunk` to `redux-saga`
- moved from `react-router`-dom to `connected-react-router` as the suggested react-router-redux has been deprecated and this library was suggested instead

### CODE REVIEW

**Comment by bneiluj** [comment]
Please move the PR into a new folder called: `cosmos-dasboard`. 
Otherwise it will populate all those files into root folder... :/

**Comment by bneiluj** [comment]
> Please move the PR into a new folder called: `cosmos-dasboard`.
> Otherwise it will populate all those files into root folder... :/

Nevermind. I see the website repo now :)

**Comment by joelb01-github** [comment]
> > Please move the PR into a new folder called: `cosmos-dasboard`.
> > Otherwise it will populate all those files into root folder... :/
> 
> Nevermind. I see the website repo now :)

ok :)

File: website/explorer/src/logo.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (bneiluj) [inline website/explorer/src/logo.svg]: can you remove this svg atom logo ?

Comment (joelb01-github) [reply website/explorer/src/logo.svg]: yes will do

**Pull request merged**